docs/cel2/notices/pectra-upgrade.md

```diff
@@ -7,7 +7,7 @@ See also the [Optimism Pectra update information](https://docs.optimism.io/build
 :::info
 This page will be updated continuously with information on upgrade runbooks and timelines as they come. This based on the last Ethereum ACDC on Jan 23rd 2025. These dates are subject to change based on how `devnet-6` goes. Ethereum L1 clients are expected to have a release out by Feb 3rd.
```

Comment:
```suggestion
This page will be updated continuously with information on upgrade runbooks and timelines as they come. This based on the last Ethereum [ACDE on Jan 30th 2025](https://ethereum-magicians.org/t/all-core-devs-execution-acde-204-january-30-2025/22594). These dates are subject to change based on how `devnet-6` goes. Ethereum L1 clients are currently scheduled to have a release out by Feb 10th.
```

I'm not sure where you got that info, but I couldn't find it in the writeup for the ACDC, anyway the more recent ACDE seems to have updated dates, or at least, there's no point telling people to update before both the consensus and execution clients are out.